mmv1/templates/terraform/examples/healthcare_dicom_store_bq_stream.tf.erb

```diff
@@ -0,0 +1,52 @@
+resource "google_healthcare_dicom_store" "default" {
+  provider = google-beta
+
+  name    = "<%= ctx[:vars]['dicom_store_name'] %>"
+  dataset = google_healthcare_dataset.dataset.id
+
+  notification_config {
+    pubsub_topic = google_pubsub_topic.topic.id
+  }
+
+  labels = {
+    label1 = "labelvalue1"
+  }
+
+  stream_configs {
+    bigquery_destination {
+      table_uri = "bq://${google_bigquery_dataset.bq_dataset.project}.${google_bigquery_dataset.bq_dataset.dataset_id}.${google_bigquery_table.bq_table.table_id}"
+    }
+  }  
+}
+
+resource "google_pubsub_topic" "topic" {
+  provider = google-beta
+
+  name     = "<%= ctx[:vars]['pubsub_topic']%>"
+}
+
+resource "google_healthcare_dataset" "dataset" {
+  provider = google-beta
+
+  name     = "<%= ctx[:vars]['dataset_name'] %>"
+  location = "us-central1"
+}
+
+resource "google_bigquery_dataset" "bq_dataset" {
+  provider = google-beta
+
+  dataset_id    = "test"
```

Comment:
Since dataset id is a unique id, it would be great to use a context var rather than a hardcoded string to make sure we don't end up with conflicts eventually (because a context var will automatically get a random suffix). For example, check out the bigquery dataset examples: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/products/bigquery/terraform.yaml#L17